tutorials/Getting_Started/README.md

```diff
@@ -90,7 +88,7 @@ compose.desktop {
     }
 }
 ```
-Then create `src/main/kotlin/main.kt` and put the following code in there:
+4. Then create `src/main/kotlin/main.kt` and put the following code in there:
```

Comment:
4. Then create `src/main/kotlin/main.kt` and add the following code there:

Reply:
done